### PR TITLE
complete flexbox task 2 stars

### PR DESCRIPTION
**Pull request by yuriy-gumenskyi**
[DEMO LINK](https://yuriy-gumenskyi.github.io/layout_stars/)

[TEST REPORT LINK](https://yuriy-gumenskyi.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"> </div>
```
Comment (loralevitska) [inline src/index.html:12]: No need empty space
```suggestion
      <div class="stars__star"></div>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+*,
+html {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  background-color: #fff;
```
Comment (loralevitska) [inline src/style.css:8]: Remove redundant styles

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  justify-content: flex-start;
+}
+
+.stars__star {
+  background-color: #fff;
```
Comment (loralevitska) [inline src/style.css:13]: Same here

**Review by loralevitska — APPROVED** [review]

File: src/style.css (hunk 2 above)
Comment (yuriy-gumenskyi) [reply src/style.css:8]: thanks for the comments! 
I see that PR already merged. Should I still adjust OR these are the comments for the future?

**Pull request closed without merging**